config/locales/en.yml

```diff
@@ -1898,6 +1898,7 @@ en:
         event_limit: "This competition limits the amount of events you can register for to %{max_events}."
       errors:
         events: "Loading Events failed, please try again"
+        bulk_update: "%{count} errors detected in request. See response payload for more information."
```

Comment:
This is only shown for API requests right? Otherwise this would be a pretty useless error message in the frontend.